tools/xabuild/XABuild.cs

```diff
@@ -121,10 +124,6 @@ static XmlDocument CreateConfig (XABuildPaths paths)
 				projectImportSearchPaths.AppendChild (searchPaths);
 			}
 
-			foreach (XmlNode property in searchPaths.SelectNodes ("property[starts-with(@name, 'MSBuildExtensionsPath')]/@value")) {
-				property.Value = string.Join (";", paths.ProjectImportSearchPaths);
```

Comment:
Maybe you should remove the elements or set the paths to `""`, to have exactly what you intended.

Reply:
And accordingly either don't create the `searchPaths` element for linux in the earlier block or create+reset.